components/brave_ads/common/pref_names.cc

```diff
@@ -38,6 +38,9 @@ const char kVersion[] = "brave.brave_ads.prefs.version";
 
 const int kCurrentVersionNumber = 4;
 
+// Ads were disabled at least once.
+const char kBraveAdsWereDisabled[] = "brave.brave_ads.were_disabled";
```

Comment:
why do we need a pref for this? Won't it get recorded as a histogram the first time it gets disabled?

Reply:
either way I think the name is confusing. I'd rather make it more verbose for clarity `kBraveAdsWereDisabledAtLeastOnce`

Reply:
This is for handling case when "ads were enabled, then disabled"